src/components/KMultiselect/KMultiselect.vue

```diff
@@ -641,6 +657,41 @@ const handleMultipleItemsSelect = (items: MultiselectItem[]) => {
   stageSelections()
 }
 
+const handleMultipleItemDeselect = (items: MultiselectItem[], restage = false) => {
```

Comment:
```suggestion
const handleMultipleItemsDeselect = (items: MultiselectItem[], restage = false) => {
```

very nit: can you change the name of the function so it's in same format as `handleMultipleItemsSelect` (`Items` in plural)